ext/soap/php_soap.h

```diff
@@ -253,4 +253,21 @@ static zend_always_inline zval *php_soap_deref(zval *zv) {
 #define Z_CLIENT_LAST_REQUEST_HEADERS_P(zv) php_soap_deref(OBJ_PROP_NUM(Z_OBJ_P(zv), 34))
 #define Z_CLIENT_LAST_RESPONSE_HEADERS_P(zv) php_soap_deref(OBJ_PROP_NUM(Z_OBJ_P(zv), 35))
 
+typedef struct {
+	php_url *url;
+	zend_object std;
+} soap_url_object;
```

Comment:
```suggestion
typedef struct soap_url_object {
	php_url *url;
	zend_object std;
} soap_url_object;
```

Can you please add a name to the struct, instead of `typedef`ing an anonymous struct? This makes the output of tools like "pahole" easier to understand. The same remark applies further below and possibly in other files as well.

Reply:
Yes, sure!